projects/swimlane/ngx-ui/src/lib/components/tabs/tabs.component.html

```diff
@@ -0,0 +1,17 @@
+<section>
+  <ul class="ngx-tabs-list list-reset" [class.tabs-vertical]="vertical" [class.tabs-horizontal]="!vertical">
+    <li *ngFor="let tab of tabs" class="ngx-tab" [class.disabled]="tab.disabled" [class.active]="tab.active">
+      <button (click)="tabClicked(tab)" [disabled]="tab.disabled">
+        <ng-container
+          *ngTemplateOutlet="labelIsString(tab.label) ? labelIsStringTmpl : tab.label; context: { $implicit: tab }"
```

Comment:
Can we do the type check inline? It should improve performance.

Reply:
updated it so the right template to use is assigned on init once